docs/sphinx/applications/python/unitary_compilation_diffusion_models.ipynb

```diff
@@ -780,7 +756,7 @@
lSpazDMAhHR0c4OjoWu527d+8CAJycnPLNc3NzK3b7hlKe+o7odcJLmUSktbwxUE+ePMHYsWPh5OQEuVyOxo0bY8uWLfnqvzpOKSIiArVq1QIArF69GjKZTPXJq3P37l2EhISgdevWqFatGuRyOVxdXTFq1Cjcv38/3zryxn7dvHkTCxYsQMOGDSGXyzF48GB88cUXkMlk2LRpk8bt+fHHHyGTyRAWFlbktmsaY5Z3yTc0NBSnTp1Cp06dYG1tDaVSiXfffVdj3fDwcABArVq1VNueV6+gMWaPHj3CJ598Ant7e1SoUAEtW7bEtm3bCo33woUL6N+/PxwdHWFhYQEXFxeMGTMGDx8+LHJbAd37zpD7msiY8YwZEenk+fPn6Ny5Mx4/fozevXsjPT0dGzZsQL9+/RAZGYnOnTsXuGzTpk0xduxYLF68GE2aNEGvXr1U8/ISkpiYGCxYsAAdO3aEh4cHzM3NcfbsWSxfvhz79u3DmTNnoFQq87U9ZswYHD9+HN26dUOPHj1QrVo19OnTB2FhYfjhhx/Qr1+/fMusXLkSZmZmGDJkSLH2ycmTJ/HVV1+hffv2+Pjjj3H27Fls374dFy9exF9//QVLS0u4uroiJCQE27dvx/nz5zF27FjY2NgAgOq/mqSnp8PX1xcXL16Ep6cnfHx8cPv2bbz//vsF7usdO3agX79+MDExQc+ePeHs7IzLly/jv//9L/bt24cTJ07A1tZWp20squ9GjBhRKvuaqNwTRGRUDh48KACIgIAAneu4uLgIAKJnz54iMzNTVX7gwAEBQPj7+6vVDwkJEQDEwYMHVWVxcXGFrj8pKUmkpaXlK1+9erUAIGbPnq1WHhAQIACIGjVqiFu3buVbrkuXLkImk4m4uDi18r/++ksAEL169dIYx6vy1vNyO3n7CYDYsGGDWv1BgwYJAGL9+vVFtpPHxcVFuLi4qJXl7cMRI0aolUdGRqrWHR4erip/8OCBUCgUonr16iI+Pl5tmfXr1wsAYvTo0WrlAISPj4/G9erSd4ba10TGjJcyiUhnCxcuhIWFhWq6Y8eOcHFxwcmTJ4vddrVq1VCpUqV85YMGDYJCocCBAwc0LjdhwgSNA9U/+eQTCCGwatUqtfIffvgBADBixIhix9yuXTu8//77amVDhw4FgGLvkzVr1sDCwgIzZ85UK/f390fHjh011k9NTUVYWBhcXFzU5vXv3x/NmjXDhg0bihVTQUpjXxOVd7yUSUQ6sbGxUY01elmNGjVw7Ngxg6xj69at+O6773DmzBk8fvwYOTk5qnl5g+df1apVK43l3bp1Q/Xq1REeHo7Q0FCYmpoiKysLa9euhbOzs0EeVdG8efN8ZTVq1AAAJCcn691uamoq4uLi0LBhQzg4OOSb37Zt23x3ch4/fhwAcOLECdy4cSPfMhkZGXjw4AEePHhg8MH9pbGvico7JmZERsbE5MWJ8tzc3ALr5M3Lq/syTeO7AMDMzKzQNrW1YMEC/Oc//0HVqlXRuXNn1KhRA1ZWVgCARYsWITMzU+Ny9vb2GstNTU0xfPhwzJgxA3v37kX37t2xbds2PHz4EKNHj9a4jbpSKBT5yszMXvy8vpxU6io1NRXAi7OImmja5kePHgEAli5dWmjbT58+NXhiVhr7mqi847eEyMjkJVaF3Z334MEDtbqlJTs7G7NmzYKjoyP++usv/Pzzz5g3bx5CQ0MREhKCrKysApeVyWQFzhs+fDhMTU2xcuVKAC8urZmYmKguN0pVXsKn6W5UAEhKSipwmYsXL0IIUeDn1cuchvK67msiqWBiRmRk6tWrBwsLC5w8eRLZ2dka6+RdkmzcuLHB129qagpA85mkBw8eICUlBZ6envnOEp06dQrPnj3Ta501atRAt27dsGfPHvzxxx+Ijo6Gv7+/5B+eqlAoUKtWLcTGxiIxMTHf/N9//z1fmYeHBwAY7LLyywrruzyv674mkgomZkRGxtLSEv369cO///6L2bNn55t/8eJF/PDDD7C2tsa7775r8PXb2tpCJpPh9u3b+eZVq1YNVlZWOHPmDNLT01Xljx8/xpgxY4q13o8//hjZ2dno27cvhBCvzUD0QYMGISsrC9OnT1cr379/v8Y3BQwZMgTW1taYOnUqLl26lG9+enq6ahyargrru5e9rvuaSAo4xozICC1YsAAnTpzAjBkzsGvXLvj4+MDS0hLXrl3Djh07IITAzz//XOjztfRVqVIltGzZEjExMRg0aBDq1q0LExMTDBo0CC4uLhg1ahQWLFiAJk2aoEePHkhNTcXevXvh4uKi8Wn52nr77bfh4uKCW7duwcHBAT169DDgVpWciRMnYuvWrVi5ciUuXbqEdu3a4fbt29i0aRO6deuG3bt3q9WvWrUq1q9fj759+6JJkyZ4++23Ub9+fWRmZiI+Ph6HDx9GmzZtEBkZqXMsRfVdntd1XxNJARMzIiNUrVo1nDx5EgsXLsT27dvx3XffISsrCw4ODujTpw/+85//4K233iqx9a9duxbjx4/Hrl27kJKSAiEEvL294eLigrCwMNjZ2SEiIgLLli2Dvb09BgwYgNDQUDRq1EjvdeYlELNnz8bgwYNVg/OlrmLFijh8+DCCg4Oxbds2nDlzBu7u7ti4cSNSUlLyJWbAi7sjz549i/nz5+PAgQOIiopCxYoVUaNGDQwZMgQffvih3vEU1nd5Xtd9TSQFMiGEKOsgiIhKQ/fu3bFnzx5cu3YNderUKetwyjXuayL9cIwZERmFy5cvY8+ePejUqRMThRLGfU2kP55fJqJybd26dbh69SrWrFkDAAgJCSnjiMov7mui4mNiRkTl2vfff4/ff/8dLi4uWLVqFdq0aVPWIZVb3NdExccxZkREREQSwTFmRERERBLBxIyIiIhIIpiYEREREUkEEzMiIiIiiWBiRkRERCQRTMyIiIiIJIKJGREREZFEMDEjIiIikoj/A4hTSZmPWH/YAAAAAElFTkSuQmCC",
```

Comment:
Just a nit because it's outside of your control: .ipynb diffs shown by GitHub are quite difficult to interpret because it shows so many changes (including things like "execution_count"), and one cannot (to my knowledge, anyway) easily compare these binary blobs to see if they are approximately similar or entirely different. 😢

Reply:
> oh :( Wondering if there is any good way to follow in order to make it a bit easy for reviewers?

Outside of asking GitHub to make a better diff viewer, I don't think there is a good way. This is just me whining about .ipynb files again. 😄